chancloser.go

```diff
@@ -83,6 +84,10 @@ type chanCloseCfg struct {
 	// forward payments.
 	disableChannel func(wire.OutPoint) error
 
+	// disconnect should be called to disconnect from the remote peer in this
```

Comment:
np, does "disconnect is called to disconnect from a peer" clearer?
I was trying to avoid a comment along the lines on "disconnect disconnects from peer", which isn't very helpful.

Reply:
Yeah, diconnect disconnects would be weird :) I'm fine with both "is called" or "can be called". Should sounds like one should do this, otherwise something may happen.